onnx/defs/generator/defs.cc

```diff
@@ -896,4 +896,115 @@ ONNX_OPERATOR_SET_SCHEMA(
           }
         }));
 
+static const char* Bernoulli_ver15_doc = R"DOC(
+Draws binary random numbers (0 or 1) from a Bernoulli distribution. The input tensor should be a tensor
+containing probabilities p (a value in the range [0,1]) to be used for drawing the binary random number,
+where an output of 1 is produced with probability p and an output of 0 is produced with probability (1-p).
```

Comment:
I suggest adding a line as below here:
```
This operator is non-deterministic and may not produce the same values in different
implementations (even if a seed is specified).
```

Reply:
I'm little bit confused. If we specified the seed, the RandomUniformLike op will generate the same value between 0 to 1. Then we can use Greater op to compare input values to product same results right?

Reply:
I think the same comment applies to RandomUniformLike. The point is: is there a guarantee that different implementations of random number generation (in GPUs vs CPUs, in different OS platforms, etc.) all use the same pseudo-random-number generator algorithm? That seems hard to guarantee. So, it is better to add the above comment. If you know that all backends use the same pseudo-random-number-generator (PRNG), please let us know what this PRNG is.